selfservice/strategy/oidc/strategy_login.go

```diff
@@ -69,6 +71,11 @@ type SubmitSelfServiceLoginFlowWithOidcMethodBody struct {
 
 	// The identity traits. This is a placeholder for the registration flow.
 	Traits json.RawMessage `json:"traits"`
+
+	// Only used in API-type flows, when an id token has been received by mobile app directly from oidc provider.
+	//
+	// required: false
+	IdToken string `json:"id_token"`
```

Comment:
In Golang, it is common to use upper-case for abbreviations, so

```suggestion
	IDToken string `json:"id_token"`
```